### PR TITLE
Database seeded, express server running

### PR DESCRIPTION
**Pull request by minglewook**

### CODE REVIEW

File: test/seed.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+for (let i = 1; i < 100; i++) {
+  thingsToSeed.push({
+    id: i, 
+    photoUrl: faker.image.imageUrl(), 
+    description: faker.lorem.sentence()
+  })
+}
+
+Photo.deleteMany((err) => {
```
Comment (RachNovo) [inline test/seed.js]: That's a cool way to do that :)

File: client/components/app.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  }
+  render() {
+    return (
+    <div>PHOTOS</div>
+    )
+  }
+
+}
+
+export default App;
```
Comment (RachNovo) [inline client/components/app.jsx]: @dekim2324 All of these files have a newline warning signs. I think that's a styling/common practice thing.

File: db/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/photos', { useNewUrlParser: true });
+
```
Comment (dekim2324) [inline db/index.js]: In the case of deployment, I suggest using the environment URI -
for example: process.env.MONGODB_URI

File: client/components/MainCarousel.jsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+
+)
```
Comment (therobinkim) [inline client/components/MainCarousel.jsx]: (Looks like a syntax error!)

File: client/components/MainPhotos.jsx [hunk 5]
```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import SinglePhoto from './SinglePhoto.jsx';
+import styled from 'styled-components';
```
Comment (therobinkim) [inline client/components/MainPhotos.jsx]: JavaScript doesn't care, but I like to import all of my npm modules first, and _then_ my own modules.

File: client/components/MainPhotos.jsx [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  left:'87%',
+  position:'relative',
+  color:'white'
+  font-size: var(--font-button-small-font-size, 14px) !important;
+  line-height: var(--font-button-small-line-height, 22px) !important;
+  letter-spacing: var(--font-button-small-letter-spacing, normal) !important;
+  font-family: var(--font-button-small-font-family, Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif) !important;
+  text-transform: var(--font-button-small-text-transform, undefined) !important;
+  padding-top: var(--font-button-small-padding-top, undefined) !important;
+  padding-bottom: var(--font-button-small-padding-bottom, undefined) !important;
```
Comment (therobinkim) [inline client/components/MainPhotos.jsx]: Are you using CSS Variables? Do we need all of these `!important` statements? In general, I do not recommend copy/pasting entire sections of CSS for this project.

File: client/components/Modal.jsx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    line-height: 24px !important;
+    letter-spacing: normal !important;
+    font-family: Circular, -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif !important;
+    padding-top: 10px !important;
+    padding-bottom: 10px !important;
+    font-weight: 800 !important;
+    padding-left: 22px !important;
+    padding-right: 22px !important;
+    color: rgb(72, 72, 72) !important;
+    text-decoration: none !important;
```
Comment (therobinkim) [inline client/components/Modal.jsx]: ...do any of these really need `!important`?

File: client/components/MainPhotos.jsx [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+)
+
+//styled components
+
+const Main = styled.div`
+  display: grid;
+  grid-template-columns: 50% 25% 25%;
+  grid-template-rows: 50% 50%;
+  border: 1px #484848
+  height: 100%
```
Comment (jkwak09) [inline client/components/MainPhotos.jsx]: Should lines these have ';' at the end as well?

**Pull request merged**